market/connector/adobe-acrobat-sign-connector/meta.json

```diff
@@ -17,6 +17,28 @@
             "groupId": "com.axonivy.connector.adobe.acrobat.sign",
             "artifactId": "adobe-acrobat-sign-connector-product",
             "type": "zip"
+        },
+        {
+            "name": "Adobe Acrobat Sign Connector",
+            "groupId": "com.axonivy.connector.adobe.acrobat.sign",
+            "artifactId": "adobe-acrobat-sign-connector",
+            "archivedArtifacts": [
```

Comment:
hey guys; you introduced a new feature for the meta.json ... without updating the the schema?
https://github.com/axonivy-market/market/blob/master/market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java
therefore you get no validation nor editing support writing these files. can you please adapt it?